src/frontend/src/lib/types/network.ts

```diff
@@ -16,18 +16,20 @@ const NetworkAppMetadataSchema = z.object({
 	explorerUrl: UrlSchema
 });
 
+const NetworkSchema = z.object({
+	id: NetworkIdSchema,
+	env: NetworkEnvironmentSchema,
+	name: z.string(),
+	icon: z.string().optional(),
```

Comment:
Not for this PR but, should we be a bit more expressive on the type of the icons?

Reply:
yes, define the type of string for icons, I will have a look

Reply:
Done with:

- PR #3875